src/interactions/event.ts

```diff
@@ -17,7 +17,7 @@ export function event(
   const _gaMeasurementId =
     process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID ?? gaMeasurementId;
 
-  if (!_gaMeasurementId || !window.gtag) {
```

Comment:
```suggestion
): void {
  if (!window.gtag) {
```

Ideal state only has this check (see PR description), same for `pageView`. No need for `gaMeasurementId?: string` on L15

Reply:
You are right, thanks @Cretezy ! could you please remove the argument? and I'll generate a new major release.

Reply:
Alright should be updated!

I did the changes on the web editor. Might be best to check locally if tests/lint/formatting still pass.